ASF-STM.user.js

```diff
@@ -564,10 +564,13 @@
             if (status === 200) {
                 errors = 0;
                 let re = /<steamID><!\[CDATA\[(.+)\]\]><\/steamID>/g;
-                username = re.exec(xhr.response)[1];
-                debugPrint(username);
-                addMatchRow(index, username);
-                callback();
+                let result = re.exec(xhr.response);
+                if (result) {
+                    username = result[1];
+                    debugPrint(username);
+                    addMatchRow(index, username);
+                    callback();
+                }
```

Comment:
This way if error happens - the script will hang. I suggest to leave `callback();` outside of `if` block

Reply:
Done